test/date-filter.js

```diff
@@ -65,9 +79,46 @@ describe("Filter by date", function() {
           links: [],
           contact_details: [],
           images: []
+        },
+        { id: 'fred-in-parliament',
+          contact_details: [],
+          links: [],
+          images: [],
+          legislative_period: 'parliament-55',
```

Comment:
I'm pretty sure that these (here and elsewhere) should be `legislative_period_id`s if they're referring to an Event defined elsewhere (rather than inlined here), in parallel to `organization_id`, `person_id`, etc